pyugrid/__init__.py

```diff
@@ -9,7 +9,8 @@
 
 from .ugrid import UGrid
 from .uvar import UVar
+from . import grid_io
 
 __version__ = '0.1.7'
 
-__all__ = ['UGrid', 'UVar']
+__all__ = ['UGrid', 'UVar', 'grid_io']
```

Comment:
I like to keep these for keeping track of things, but I can remove if you want.  No strong opinions about it.